src/Applications/LDAS_App/ldas_setup

```diff
@@ -300,9 +312,13 @@ class LDASsetup:
 
         if  int(self.rqdExeInp['LSM_CHOICE']) == 1 :
             self.catch = 'catch'
-        else :
+            self.clmv       = ''
+        if  int(self.rqdExeInp['LSM_CHOICE']) == 2 :
             self.catch = 'catchcn'
-
+            self.clmv  = '40'
```

Comment:
Across this PR and (primarily) the matching GEOSgcm_GridComp PR, the model naming convention could be a bit more consistent.  Here, we use "catchcn40" and "catchcn45".
The LAND_PARAMS values are "CN_CLM4" and "CN_CLM45".
The directories in the src code tree are "GEOScatchCNCLM40" and "GEOScatchCNCLM45".
Why not stick to one core syntax, say "CNCLM40" and "CNCLM45"?

Reply:
Made changes (mostly to GEOSgcm_GridComp) to ensure that naming convention is consistent across parts of the model that the user sees, i.e., when prompted fro LSM_CHOICE and for the LAND_PARAMS collections. Internally, there are still some different names that are used.